package-lock.json

```diff
@@ -0,0 +1,17852 @@
+{
```

Comment:
Do you want this back?

Reply:
No, I hadn't confirmed if it was in .gitignore, I have added it now